go.mod

```diff
@@ -124,6 +126,6 @@ require (
 
 replace (
 	github.com/docker/docker => github.com/docker/docker v20.10.3-0.20211208011758-87521affb077+incompatible
-	go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace => github.com/tonistiigi/opentelemetry-go-contrib/instrumentation/net/http/httptrace/otelhttptrace v0.0.0-20211026174723-2f82a1e0c997
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => github.com/tonistiigi/opentelemetry-go-contrib/instrumentation/net/http/otelhttp v0.0.0-20211026174723-2f82a1e0c997
+	go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace => go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace v0.27.1-0.20211201163413-2bb24f926b80
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.28.1-0.20220111222512-7e31ebe04030
 )
```

Comment:
These lines no longer need to be in the `replace()` section

Reply:
We need them for @tonistiigi changes from upstream that are not yet released so we can avoid his own fork:
* https://github.com/open-telemetry/opentelemetry-go-contrib/commit/2bb24f926b80e0d13e278cbeed85f4d10ffbccb3
* https://github.com/open-telemetry/opentelemetry-go-contrib/commit/7e31ebe040306aee2c826972269f938f9f0e7c34

Reply:
Can't you just use these commits in regular require block now without replace?

Reply:
Hum yes actually